### DISCUSSION TITLE
Custom sidebar category name

### DISCUSSION THREAD

**Discussion by peterparker2005** [payloadcms/payload#9940] [Q&A] [answered]
I found a screenshot at https://payloadcms.com/use-cases/enterprise-app-builder where you can see Company / Admin blocks. How is it real to redefine names and split collections / globals into this type of structure?

<img width="851" alt="image" src="https://github.com/user-attachments/assets/06d4e94f-52d3-4870-9a21-b7a5e1c484cb" />

**Comment by akhrarovsaid** [comment, marked as answer]
You can do this with the `admin.group` property of your global or collection config.

See here for [collections](https://payloadcms.com/docs/admin/collections#admin-options), and here for [globals](https://payloadcms.com/docs/admin/globals#admin-options).
